src/app/components/classes/classes.ts

```diff
@@ -1,13 +1,14 @@
 import { ICustomer } from "src/app/interfaces/interfaces";
 
 export class Customer implements ICustomer {
-    constructor(id: number, name: string, phoneNumber: string, email: string, password: string, birth: Date ) {
+    constructor(id: number, name: string, phoneNumber: string, email: string,  birth: Date, password: string, hasSubscription: boolean ) {
         this.id = id;
         this.name = name,
         this.phoneNumber = phoneNumber,
         this.email = email,
-        this._password = password,
         this.birth = birth,
+        this._password = password,
```

Comment:
Usually, private varibales (from _...) uses with getters and setters. But, it's not. Also not as another